bindings/c/meshIOc.cpp

```diff
@@ -52,15 +52,10 @@ void manifold_material_set_metalness(ManifoldMaterial *mat, double metalness) {
   from_c(mat)->metalness = metalness;
 }
 
-void manifold_material_set_color(ManifoldMaterial *mat, ManifoldVec4 color) {
+void manifold_material_set_color(ManifoldMaterial *mat, ManifoldVec3 color) {
```

Comment:
Just taking a look on my phone from the hospital with a suddenly one month early (healthy) new born daughter, but this looks ok to me. The `from_c` overload should be doing its job here so nothing needs to change past the type.

Reply:
Thanks, and congrats! My first daughter was also a month early - had to spend a week in the NICU, which wasn't much fun. But she's fine and it's hard to believe she's ten now. Good job on getting out of grad school - want to share your dissertation? Enjoy your family time, and let me know if you want a Google referral.